breathecode/authenticate/tests/urls/tests_academy_id_member.py

```diff
@@ -469,3 +469,155 @@ def test_academy_id_member_with_two_roles(self):
+        profile_academy = self.get_profile_academy(1)
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(self.all_profile_academy_dict(), [{
+            'academy_id': 1,
+            'address': None,
+            'email': None,
+            'first_name': None,
+            'id': 1,
+            'last_name': None,
+            'phone': '',
+            'role_id': role,
+            'status': 'INVITED',
+            'user_id': 2,
+        }])
+
+    def test_academy_id_member_post_no_invite(self):
+        """Test /academy/:id/member"""
+        role = 'konan'
+        model = self.generate_models(authenticate=True, role=role,
+            capability='crud_member', profile_academy=True)
+        url = reverse_lazy('authenticate:academy_id_member', kwargs={'academy_id':1})
+        data = {'role': role, 'invite': True}
+        response = self.client.post(url, data)
+        json = response.json()
+        print(json)
+        expected = {'detail': "Please specify user id or member email",
+            'status_code' : 400}
+        profile_academy = self.get_profile_academy(1)
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(self.all_profile_academy_dict(), [{
+            'academy_id': 1,
+            'address': None,
+            'email': None,
+            'first_name': None,
+            'id': 1,
+            'last_name': None,
+            'phone': '',
+            'role_id': role,
+            'status': 'INVITED',
+            'user_id': 2,
+        }])
+    
+    def test_academy_id_member_post_user_with_not_student_role(self):
+        """Test /academy/:id/member"""
+        role = 'konan'
+        model = self.generate_models(authenticate=True, role=role,
+            capability='crud_member', profile_academy=True)
+        url = reverse_lazy('authenticate:academy_id_member', kwargs={'academy_id':1})
+        data = { 'role': role , 'user' : model['user'].id, 'first_name': 'Kenny', 'last_name': 'McKornick'}
+        response = self.client.post(url, data)
+        json = response.json()
+        print(json)
+        expected = {'detail': f'This user is already a member of this academy as {role} ({role})',
+            'status_code' : 400}
+        profile_academy = self.get_profile_academy(1)
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(self.all_profile_academy_dict(), [{
+            'academy_id': 1,
+            'address': None,
+            'email': None,
+            'first_name': None,
+            'id': 1,
+            'last_name': None,
+            'phone': '',
+            'role_id': role,
+            'status': 'INVITED',
+            'user_id': 2,
+        }])
+
+    def test_academy_id_member_post_user_with_student_role(self):
```

Comment:
We need a new test like it, the current profile as student and the new as teacher